ocaml/idl/datamodel_pool.ml

```diff
@@ -1331,6 +1368,17 @@ let t =
             "coordinator_bias"
             "true if bias against pool master when scheduling vms is enabled, \
              false otherwise"
+        ; field ~lifecycle:[] ~qualifier:DynamicRO ~ty:(Ref _secret)
+            ~default_value:(Some (VRef null_ref)) "telemetry_uuid"
+            "The UUID of the pool for identification of telemetry data"
+        ; field ~lifecycle:[] ~qualifier:DynamicRO ~ty:telemetry_frequency
+            ~default_value:(Some (VEnum "weekly")) "telemetry_interval"
+            "The frequency of telemetry collection to be carried out"
+        ; field ~qualifier:DynamicRO ~lifecycle:[] ~ty:DateTime
+            ~default_value:(Some (VDateTime Date.epoch))
+            "telemetry_next_schedule"
```

Comment:
That is why it is named as `next_schedule`. The time when the next telemetry task will run is not a regular one. It may be adjusted based on the submitting result.
What's more, the time is not an exact point the telemetry task will be carried out. The telemetry function is triggered by a regular timer, and carries out telemetry task only if the current timestamp is >= the `next_schedule`.